versioned_docs/version-11.x.x/development_suite/api-console/advanced-section/api-console-configuration.md

```diff
@@ -107,3 +107,27 @@ For example, to configure the `microservice-gateway` with 5 replicas, the config
   }
 }
 ```
+
+## Add custom environment variables
```

Comment:
Why adding in v11 doc as well? I would not change previous version docs unless there's a strong reason for it

Reply:
I added this information on the v11 too because this example is valid for the v11 as well.

If you prefer I can remove that, but can be misleading for a customer who uses the v11 and looks for this specific information.

Tell me if you prefer to remove this example from the v11.